crypto/key_manager.go

```diff
@@ -38,9 +38,9 @@ type KeyManager interface {
 	// manager.
 	Signer() (crypto.Signer, error)
 	// SignatureAlgorithm returns the value that identifies the signature algorithm.
-	SignatureAlgorithm() trillian.SignatureAlgorithm
+	SignatureAlgorithm() sigpb.DigitallySigned_SignatureAlgorithm
 	// HashAlgorithm returns the type of hash that will be used for signing with this key.
-	HashAlgorithm() trillian.HashAlgorithm
+	HashAlgorithm() crypto.Hash
```

Comment:
This doesn't return an algorithm any more so is potentially confusing. Can you TODO it for rework later.